examples/named_entity_recognition/Named_Entity_Recognition_with_Tensorflow_and_🔭_Galileo.ipynb

```diff
@@ -23,6 +23,14 @@
       "outputs": [],
```

Comment:
<p>Same, we try to hide that ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rungalileo/examples/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>